src/lib/assetBridger/erc20Bridger.ts

```diff
@@ -696,6 +788,11 @@ export class Erc20Bridger extends AssetBridger<
       // in the future we want to do proper estimation here
       /* eslint-disable @typescript-eslint/no-unused-vars */
       estimateL1GasLimit: async (l1Provider: Provider) => {
+        if (!this.nativeTokenIsEth) {
+          // measured 172867 - add some padding
+          return BigNumber.from(200000)
```

Comment:
this probably won't work for L3->L2 withdrawal

Reply:
bumped in https://github.com/OffchainLabs/arbitrum-sdk/pull/310/commits/af5586fc54fefc204d86f0eceef671f42ebd8fcf